Assets/Scripts/Layers/LayerTypes/CartesianTileBuildingColorLayerGameObject.cs

```diff
@@ -88,5 +120,13 @@ private void AddLayersRecursive(LayerData layer, List<DatasetLayerGameObject> li
                 AddLayersRecursive(child, list);
             }
         }
+
+        public void LoadProperties(List<LayerPropertyData> properties)
+        {
+            var propertyData = properties.OfType<CartesianTileBuildingColorPropertyData>().FirstOrDefault();
+            if (propertyData == null) return;
+
+            this.propertyData = propertyData;
```

Comment:
hier set je alleen de property, moet die data nog geladen worden?

Reply:
Het laden van de CSV gebeurd in de start methode zodat het niet uitmaakt hoe de data hier aankwam, of je nu een nieuwe laag aanmaakt of laad. Dat zie je hier: https://github.com/Netherlands3D/twin/pull/238/files/3bbde5981d212064d8bc274b69b07691bab96f94#diff-faa000c2270eab634c6656cb7498446e2403481e5634d41416da870251fbf823R32

Reply:
kan dit als comment erbij gezet worden voor toekomstige devs zodat ze weten waar de data vandaan komt/ waar de ingeladen property wordt ingeladen